WeakAuras/WeakAuras.lua

```diff
@@ -30,6 +30,7 @@ function WeakAurasTimers:ScheduleTimerFixed(func, delay, ...)
 end
 
 local LDB = LibStub:GetLibrary("LibDataBroker-1.1")
+local LDBIcon = LibStub("LibDBIcon-1.0")
```

Comment:
Don't we need to bundle something to enable that library?

Reply:
Yes we need to bundle the library to make it work, btw here i used LibStub("LibDBIcon-1.0") instead of  local LDB = LibStub:GetLibrary("LibDBIcon-1.0") i have no idea if there is any difference, but i will probably update this.

Reply:
Ah take a look at commits to https://github.com/WeakAuras/WeakAuras2/commits/master/.pkgmeta on how to tell the packager to bundle libaries.